test/StakeManager.t.sol

```diff
@@ -245,6 +294,76 @@ contract ExecuteAccountTest is StakeManagerTest {
+        vm.stopPrank();
+    }
+
+    function test_ExecuteAccountMintMP() public {
+        uint256 stakeAmount = 10_000_000;
+        deal(stakeToken, testUser, stakeAmount);
+
+        userVaults.push(_createStakingAccount(makeAddr("testUser"), stakeAmount, 0));
+        userVaults.push(_createStakingAccount(makeAddr("testUser2"), stakeAmount, 0));
+        userVaults.push(_createStakingAccount(makeAddr("testUser3"), stakeAmount, 0));
+
+        console.log("######### NOW", block.timestamp);
+        console.log("# START EPOCH", stakeManager.currentEpoch());
+        console.log("# PND_REWARDS", stakeManager.pendingReward());
+
+        for (uint256 i = 0; i < 3; i++) {
+            deal(stakeToken, address(stakeManager), 100 ether);
+            vm.warp(stakeManager.epochEnd());
+            console.log("######### NOW", block.timestamp);
+            stakeManager.executeEpoch();
+            console.log("##### NEW EPOCH", stakeManager.currentEpoch());
+            console.log("# PND_REWARDS", stakeManager.pendingReward());
+
+            for (uint256 j = 0; j < userVaults.length; j++) {
+                (address rewardAddress,,, uint256 currentMPBefore, uint256 lastMintBefore,, uint256 epochBefore) =
+                    stakeManager.accounts(address(userVaults[j]));
+                uint256 rewardsBefore = ERC20(stakeToken).balanceOf(rewardAddress);
+                console.log("-Vault number", j);
+                console.log("--=====BEFORE=====");
+                console.log("---### currentMP :", currentMPBefore);
+                console.log("---#### lastMint :", lastMintBefore);
+                console.log("---## user_epoch :", epochBefore);
+                console.log("---##### rewards :", rewardsBefore);
+                console.log("--=====AFTER======");
+                stakeManager.executeAccount(address(userVaults[j]), epochBefore + 1);
+                (,,, uint256 currentMP, uint256 lastMint,, uint256 epoch) =
+                    stakeManager.accounts(address(userVaults[j]));
+                uint256 rewards = ERC20(stakeToken).balanceOf(rewardAddress);
+                console.log("---### deltaTime :", lastMint - lastMintBefore);
+                console.log("---### currentMP :", currentMP);
+                console.log("---#### lastMint :", lastMint);
+                console.log("---## user_epoch :", epoch);
+                console.log("---##### rewards :", rewards);
+                console.log("--=======#=======");
+                console.log("--# TOTAL_SUPPLY", stakeManager.totalSupply());
+                console.log("--# PND_REWARDS", stakeManager.pendingReward());
+                assertEq(lastMint, lastMintBefore + stakeManager.EPOCH_SIZE(), "must increaase lastMint");
+                assertEq(epoch, epochBefore + 1, "must increase epoch");
+                assertGt(currentMP, currentMPBefore, "must increase MPs");
+                assertGt(rewards, rewardsBefore, "must increase rewards");
```

Comment:
These are great.

I wonder if we can make these assertions stronger. 
Do you think it's feasible to also assert by how much MPs and rewards increase?

Or maybe a rough ballpark, so that one can say "it must increase by at least this amount" ?